server/server.go

```diff
@@ -543,3 +543,8 @@ func (s *Server) Shutdown(ctx context.Context) error {
 func (s *Server) GetEcho() *echo.Echo {
 	return s.e
 }
+
+// GetProfile returns the server's profile.
```

Comment:
This is only used in the test. We should get the profile or data directory from setUpForPITRTest().